### ISSUE TITLE
[Feat] Add a sale pop up

### ISSUE THREAD

**Issue by Amulya-B28** [mdazfar2/Ezyshop#387]
### Description

add a sale pop up which appears when the website is loaded and can be close by a close button.

### Screenshots

![image](https://github.com/user-attachments/assets/4ba352c4-9f47-4f98-8a8a-1f727a408a7d)

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/mdazfar2/Ezyshop/blob/main/CONTRIBUTING.md)
- [X] I'm a GSSoC'24-Extd contributor
- [X] I'm a Hacktoberfest'24 contributor
- [X] I am willing to work on this issue (optional)

**Comment by mdazfar2** [comment]
Hello @Amulya-B28, this is a very good feature, but we are converting the entire code into the Next.js framework for easier backend integration. Once the Next.js conversion is complete, I will let you know so you can work on that issue.

If you're interested in contributing to our Next.js code, please let me know. We've completed our landing page in Next.js, but it doesn't match the current one in terms of text style, font, and several other things.

* The whole Next.js code is in the `Benjamin` directory within the Ezyshop root folder. Below, I provide the steps to run the Next.js project.


```bash
 cd Benjamin
```

```bash
 npm i
```


```bash
 npm run dev
```

* Then run in your browser `localhost:3000`


* Do changes what you want but make sure it is full duplicate of HTML, CSS website which is now deployed.


***Thank you!***

**Comment by mdazfar2** [comment]
Hey @Amulya-B28, sorry for the late reply. I’ve been a little busy due to my health condition. We have converted the entire code to Next.js and Tailwind CSS for better backend integration. You raised a good issue, but it was based on the old HTML and CSS version. We’ve moved everything to Next.js and Tailwind CSS. Please look into that, and don’t forget to join our Discord for the latest website announcements and features.

- Please don't be discouraged. I will close the issue for now, but you can raise a new one that is truly needed for our website. Thank you.
